datanode/gateway/graphql/schema.graphql

```diff
@@ -4333,7 +4333,7 @@ type NodeSignatureEdge {
 "Connection type for retrieving cursor-based paginated node signature information"
 type NodeSignaturesConnection {
   "List of node signatures available for the connection"
-  edges: [NodeSignatureEdge]
+  edges: [NodeSignatureEdge]!
```

Comment:
is it possible for this this be `edges: [NodeSignatureEdge!]!` ?

Reply:
Sure, I'll make those updates for you.